churaverse-plugins-client/src/synchroBreakPlugin/ui/timeLimitFormContainer/timeLimitFormContainer.ts

```diff
@@ -0,0 +1,95 @@
+import { Store, IMainScene, DomManager, makeLayerHigherTemporary, domLayerSetting } from 'churaverse-engine-client'
+import { NetworkPluginStore } from '@churaverse/network-plugin-client/store/defNetworkPluginStore'
+import { IGameUiComponent } from '@churaverse/game-plugin-client/interface/IGameUiComponent'
+import '@churaverse/player-plugin-client/store/defPlayerPluginStore'
+import '@churaverse/game-plugin-client/gameUiManager'
+import { TimeLimitForm } from './component/TimeLimitForm'
+import { TimeLimitConfirmMessage } from '../../message/timeLimitConfirmMessage'
+
+// 制限時間入力部分
+export const TIME_LIMIT_INPUT_FIELD_ID = 'time-limit-input-field'
```

Comment:
idはグローバルな値なので、名前空間を分けて衝突を避ける意味で`synchro-break`をプレフィックスにつけた方が良さそうです
```suggestion
export const TIME_LIMIT_INPUT_FIELD_ID = 'synchro-break-time-limit-input-field'
```